stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/extensions/Channel.kt

```diff
@@ -29,3 +33,40 @@ private fun Channel.includesCurrentUser(): Boolean {
     val currentUserId = ChatClient.instance().getCurrentUser()?.id ?: return false
     return members.any { it.user.id == currentUserId }
 }
+
+/**
+ * Returns the channel name if exists, or the list of member names if the channel is distinct.
+ *
+ * @param context The context to load string resources.
+ * @param currentUser The currently logged-in user.
+ * @param fallback The resource identifier of a fallback string if the [Channel] object lacks
+ * @param maxMembers The maximum number of members used to generate a name for a distinct channel.
+ * information to construct a valid display name string.
+ *
+ * @return The display name of the channel.
+ */
+@InternalStreamChatApi
+@JvmOverloads
```

Comment:
Should we have JVM overloads if we have `InternalStreamChatApi`?

I imagine it's useful in case people opt into this and want to use it in Java, but it might look a bit confusing.

Reply:
👍 I think we can safely remove it